### DISCUSSION TITLE
Welcome to TcUnit-VsExtension Discussions!

### DISCUSSION THREAD

**Discussion by philippleidig** [philippleidig/TcUnit-VsExtension#3] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by runtimevic** [comment]
Hello, 
is possible install TcUnit Visual Studio Extension?
thanks Víctor.

**Reply by philippleidig** [reply]
@runtimevic, please see latest release
